hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3ADataBlocks.java

```diff
@@ -798,6 +800,8 @@ public String toString() {
    * Buffer blocks to disk.
    */
   static class DiskBlockFactory extends BlockFactory {
+    private static final String ESCAPED_FORWARD_SLASH = "EFS";
```

Comment:
think I might prefer something more distinguishable from text, e.g "_FS_" and "_BS_" so its easier to read in a dir listing